client/src/pages/PaymentPage.js

```diff
@@ -1,10 +1,7 @@
 import React, { Component } from 'react';
-import PrimaryNavbar from './../components/Navbar';
+import { PrimaryNavBar, PaymentPageContent, PaymentPageProgress } from './../components';
```

Comment:
Here. It says PrimaryNavBar. But on line 36 we use PrimaryNavbar

Reply:
It's changed to PrimaryNavBar in my code...

Reply:
Wait did you push all the changes then? Because opening the file on github shows what I'm seeing.

Reply:
That's so weird, I thought I pushed it, but I'll fix it.